CheatSheets/git-cheatsheet.md

```diff
@@ -36,10 +37,13 @@ Table of Contents
 | `git diff` | Compare and show updated codes |
 | `git add [file name]` | Add file changes in next commit |
 | `git add .` | Add all unstaged changes in next commit |
+| `git reset [file name]` | Used to unstage the staged files |
+| `git clean -f` | To delete or remove unstaged files forcefully |
 | `git commit -m "message about updates"` | Commit changes to current branch |
 | `git rm [file]` | Deletes the file from your working directory and stages the deletion |
 | `git pull` | Fetches and merges changes on the remote server to your working directory |
 | `git fetch` | Gathers remote commits but does not merge them unlike `pull` |
+| `git mv ___(file name) ___(file name to be renamed)` | This is used to rename the files |
```

Comment:
| `git mv [file name] [file name to be renamed]` | This is used to rename the files |